src/scenes/ContactsPage/components/Form/Form.js

```diff
@@ -173,7 +173,7 @@ const Form = () => {
             ></input>
             <div className={styles.attachmentName}>
               <button
-                type="button"
+                type="reset"
```

Comment:
why do we need to change button type?

Reply:
look this task [Contacts] Attach file bug
https://app.clickup.com/t/g58hfc